Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -236,6 +236,13 @@ final class MathOperatorTests: XCTestCase {
       XCTAssertEqual(y, expected)
     }
 
+    func testElu() {
+      let x = Tensor<Float>([-1.0, 2.0, 3.0])
```

Comment:
Please use 4-space indentation. Same for `testSoftPlus()` and `testSoftSign()` above.